mteb/abstasks/AbsTaskClassification.py

```diff
@@ -75,15 +76,31 @@ def __init__(
         **kwargs,
     ):
         super().__init__(**kwargs)
+        if method != "logReg":
+            warnings.warn(
+                "Passing method to AbsTaskClassification is deprecated and will be removed in v2.0.0.",
```

Comment:
Using this quotation to denote parameters. Can apply to the rest.
```suggestion
                "Passing `method` to AbsTaskClassification is deprecated and will be removed in v2.0.0.",
```